components/InputText.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { Text, TextInput, View } from 'react-native';
+
+export class InputText extends React.Component {
+
+	constructor(props) {
+    super(props);
+    this.state = {text: ''};
+  }
+
+  render() {
```

Comment:
Please use appropriate indentations to make the code more readable.

Reply:
In local, indentation is pretty but it collapse in github. Added lint and prettier to fix indentation in github, pls check next pull req whether it is work or not.